src/app/tests/suites/certification/Test_TC_ICDM_5_2.yaml

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
+
+name: 217.2.8 [TC-ICDM-5.2] Operating Mode with DUT as Server - Multi-Fabrics
+
+PICS:
+    - ICDM.S
+    - ICDM.S.F02
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic Information"
+    endpoint: 0
+
+tests:
+    - label:
+          "Step 1a: TH1 reads from the DUT the RegisteredClients attribute.
+          RegisteredClients is empty."
+      verification: |
+          TH1 reads the RegisteredClients attribute from the DUT. If a RegisteredClients entry is present, TH sends an unregisterclient command and ensures that the RegisteredClients attribute is empty.
+
+          ./chip-tool icdmanagement read registered-clients 1 0
+          [1718188779.575015][1538:1540] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_0046 Attribute 0x0000_0003 DataVersion: 2743878261
+          [1718188779.577470][1538:1540] CHIP:TOO:   RegisteredClients: 0 entries
+          [1718188779.577928][1538:1540] CHIP:EM: <<< [E:53088i S:19826 M:50965508 (Ack:261929127)] (S) Msg TX to 1:0000000000000001 [7914] [UDP:[fe80::e65f:1ff:fe49:ae1a%eth0]:5540] --- Type 0000:10 (SecureChannel:StandaloneAck)
+
+
+          If any entries are present, please send the command below to remove them:
+
+          ./chip-tool icdmanagement unregister-client 1<checkinnodeid> 1 0
+      disabled: true
```

Comment:
This seems to be adding new manual tests.

new manual test addition is not allowed anymore.

Reply:
some of these are old tests (<=1.3 release) that were manual, until those are automated we have to keep/update the manual tests.

Reply:
This is a new file containing manual tests. This will not be accepted in the SDK.